common/macresman.cpp

```diff
@@ -320,15 +315,10 @@ bool MacResManager::isMacBinary(SeekableReadStream &stream) {
 		//uint32 rsrcSizePad = (((rsrcSize + 127) >> 7) << 7);
 
 		// Length check
-		if (MBI_INFOHDR + dataSizePad + rsrcSize <= (uint32)stream.size()) {
-			resForkOffset = MBI_INFOHDR + dataSizePad;
-		}
+		if (MBI_INFOHDR + dataSizePad + rsrcSize == (uint32)stream.size())
```

Comment:
That test is incorrect because with that change only macbinary files created with ISOBuster will (or at least might) work.

A test that might work for both ISOBuster and standard macbinary files would be:
```c++
if (MBI_INFOHDR + dataSizePad + rsrcSize == (uint32)stream.size() || MBI_INFOHDR + dataSizePad + rsrcSizePad == (uint32)stream.size())
```

Reply:
Indeed, this just breaks things. What is the reasoning behind this change?